System.Web.Mvc.Expressions/Internals/MvcExtensions.cs

```diff
@@ -20,14 +21,14 @@ public static string GetActionName(this LambdaExpression actionExpression)
             }
 
             var actionMethod = methodCallExpression.Method;
+            string result;
+            if (ActionNameInfo.TryGetValue(actionMethod, out result))
+                return result;
```

Comment:
Sorry, didn't notice you always do curly brackets.  In all software I do, I avoid any ceremony when I can..such as these.